resources/certs/ca-cert

```diff
@@ -1,20 +1,20 @@
 -----BEGIN CERTIFICATE-----
-MIIDRjCCAi4CCQCZ2pXVwpC9gzANBgkqhkiG9w0BAQsFADBlMQswCQYDVQQGEwJV
```

Comment:
Why did we change the CA cert?

Reply:
The TLS tests did not work without updating the certs.  Assumed they are expired, but can put them back to be regened locally for each user.

Reply:
reset back to upstream/master for net no change

Reply:
changed, commited, and squashed

Reply:
TLS test were not working so I rebuild them locally.
Reset back to master
commited and squashed
net no change